libs/src/api/user.js

```diff
@@ -629,10 +641,19 @@ class Users extends Base {
       addOps.push(this.contracts.UserFactoryClient.updateIsCreator(userId, metadata[USER_PROP_NAME_CONSTANTS.IS_CREATOR]))
     }
 
-    // Execute update promises concurrently
-    // TODO - what if one or more of these fails?
-    const ops = await Promise.all(addOps)
-    return { ops: ops, latestBlockNumber: Math.max(...ops.map(op => op.txReceipt.blockNumber)) }
+    let ops; let latestBlockNumber = -Infinity; let latestBlockHash
+    if (addOps.length > 0) {
```

Comment:
could ops.length be 0? mabye we should throw/return an error if this if doesn't get hit

Reply:
not sure if that's the right thing here... this one is addUserOps, but I know that in updateUserOps, ops would be empty for wallet addition/removal and we wouldn't want to throw there